sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverClient.java

```diff
@@ -492,7 +494,16 @@ IterableStream<ServiceBusReceivedMessage> peekMessages(int maxMessages, long seq
      * Receives an iterable stream of {@link ServiceBusReceivedMessage messages} from the Service Bus entity. The
      * receive operation will wait for a default 1 minute for receiving a message before it times out. You can
      * override it by using {@link #receiveMessages(int, Duration)}.
-     *
+     * <p>
+     * The 1-minute timeout is a client-side feature. Each time the application calls receiveMessages, a timer is started
```

Comment:
personal: when I am mentioning code invocations, I try to use `{@code receiveMessages}` or `{@link }` so users know it's a method or something they can call.